### PR TITLE
feat: create user profile group component

### PR DESCRIPTION
**Pull request by fampinheiro**

### CODE REVIEW

File: packages/react-components/src/organisms/UserProfileGroups.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    <Card>
+      <div
+        css={{
+          display: 'flex',
+          flexDirection: 'column',
+        }}
+      >
+        <Headline2 styleAsHeading={3}>{firstName}’ Groups</Headline2>
+        <Paragraph accent="lead">
+          {firstName}’ team is collaborating with other teams via groups, which
```
Comment (jeysal) [inline packages/react-components/src/organisms/UserProfileGroups.tsx:87]:
```suggestion
          {firstName}’s team is collaborating with other teams via groups, which
```
I assume?

File: packages/react-components/src/organisms/UserProfileGroups.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    [`&:not(:first-of-type) > :nth-child(odd)`]: {
+      display: 'none',
+    },
+  },
+});
+
+type UserProfileGroupsProps = Pick<UserResponse, 'firstName'> & {
+  groups: ReadonlyArray<{
+    name: string;
+    role: 'Leader' | 'Member';
```
Comment (jeysal) [inline packages/react-components/src/organisms/UserProfileGroups.tsx:52]: It feels like this should be defined somewhere in model?

File: packages/react-components/src/organisms/UserProfileGroups.tsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  gridColumnGap: `${18 / perRem}em`,
+
+  margin: 0,
+  marginTop: `${24 / perRem}em`,
+
+  padding: 0,
+  listStyle: 'none',
+});
+
+const listItemStyle = css({
```
Comment (jeysal) [inline packages/react-components/src/organisms/UserProfileGroups.tsx:28]: nice grid m8

**Review by jeysal — COMMENTED** [review]
mostly lgtm after I understood (umm, somewhat) the grid ^^

File: packages/react-components/src/organisms/UserProfileGroups.tsx (hunk 2 above)
Comment (fampinheiro) [reply packages/react-components/src/organisms/UserProfileGroups.tsx:52]: I'm clarifying this with James. The designs don't use the same roles as the group page, that's why they differ.
Leader is someone that is a Chair or Project Manager in a group. Member is someone that is part of a team that is part of a group.

**Review by jeysal — APPROVED** [review]

File: packages/react-components/src/organisms/UserProfileGroups.tsx (hunk 2 above)
Comment (fampinheiro) [reply packages/react-components/src/organisms/UserProfileGroups.tsx]: Thanks for pointing this out. The requirement was to have the group roles extended with Member.
The "member" role only exists on the frontend, and it's inferred from the group response. Do you still believe the type definition should be elsewhere?

**Pull request merged**